packages/react-native-reanimated/src/layoutReanimation/animationsManager.ts

```diff
@@ -98,7 +101,11 @@ function createLayoutAnimationManager() {
       if (!value) {
         return;
       }
-      stopObservingProgress(tag, value);
+      if (IS_FABRIC) {
+        value.removeListener(tag + TAG_OFFSET);
```

Comment:
The `stop` function is only called from native, so having it call the `endLayoutAnimation` function back is a bit redundant, since I do all the necessary cleanup when I cancel animations. But I think we can change it back, since the `endLayoutAnimation` should return immediately, if there is no animation configured.